RoundMaster/RoundMaster.js

```diff
@@ -153,15 +157,16 @@
  * v4.045  07/07/2023 Additional effects added. Added "always" parameter to --start which forces the 
  *                    tracker into an active state. Updated menu colours to be more readable.
  *                    Fixed tracker rotation status to be saved properly between sessions.
+ * v4.046  08/09/2023 Added error handling based on The Aaron's technique. Ensured switch statements are
+ *                    based on consistent case of characters.
  **/
  
 var RoundMaster = (function() {
 	'use strict'; 
-	var version = 4.045,
+	var version = 5.046,
```

Comment:
After reviewing this, is this supposed to be `4.046`?

Reply:
5.046 is correct. This release is a major version change for RoundMaster (major error handling change), and for RoundMaster the Major and Minor version numbers increment independently (historical reasons...). OK to release as is. Thanks